e2e/qgb/deployer_test.go

```diff
@@ -38,9 +38,9 @@ func TestDeployer(t *testing.T) {
 	HandleNetworkError(t, network, err, false)
 
 	// FIXME should we use the evm client here or go for raw queries?
-	evmClient := orchestrator.NewEvmClient(nil, bridge, nil, network.EVMRPC)
+	evmClient := orchestrator.NewEvmClient(nil, bridge, nil, network.EVMRPC, orchestrator.DEFAULTEVMGASLIMIT)
 
 	eventNonce, err := evmClient.StateLastEventNonce(&bind.CallOpts{Context: network.Context})
 	assert.NoError(t, err)
-	assert.Equal(t, uint64(0), eventNonce)
+	assert.Equal(t, uint64(1), eventNonce)
```

Comment:
we might want to document the reasoning behind this change somewhere

Reply:
nvm, just saw #558

Reply:
here https://github.com/celestiaorg/celestia-app/issues/373#issuecomment-1193336317